cmd/update-main.go

```diff
@@ -62,15 +63,22 @@ EXAMPLES:
 `,
 }
 
-const releaseTagTimeLayout = "2006-01-02T15-04-05Z"
-
-const minioReleaseURL = "https://dl.minio.io/server/minio/release/" + runtime.GOOS + "-" + runtime.GOARCH + "/"
+const (
+	minioReleaseTagFormat = "2006-01-02T15-04-05Z"
```

Comment:
That is just a variable name - const are called what they really are - RFC3339 not RFC3339Layout.